tests/Tester.py

```diff
@@ -66,6 +66,7 @@ def setUpClass(cls):
         ROOT_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
         directory = os.path.join(ROOT_DIR, 'grammars')
 
+        # should be kept in sync with .github/workflows/generate-snippet-hashes.yml
```

Comment:
I am not sure I love this comment, I think what you mean to say is: these files *are* kept in sync with .github/workflows/generate-snippet-hashes.yml, is that correct? Then maybe we can document the mechanism here through which that happens (in the on-build script, right?)

Reply:
Oh sorry for the confusion. The tests/Tester.py file compiles a list of files that affect how hedy works, to determine if a test can be skipped. Any change to any of these files will cause the entire snippet file to be invalidated.

This list is now also included in the `.github/worksflows/generate-snippet-hashes.yml` file, to detect if we should regenerate the cached file. So what I meant with the comment is that we manually have to keep the list in sync. Any time we change the "set of files that determine how hedy works" we have to update in two places.

Let me know if that makes sense and/or whether the comment needs more explanation.